src/signals/incident/containers/KtoContainer/components/KtoForm/index.js

```diff
@@ -215,23 +217,28 @@ const KtoForm = ({ options, isSatisfied, onSubmit }) => {
         />
       </GridArea>
 
-      <GridArea>
-        <StyledLabel id="subtitle-allows-contact">
-          Mogen wij contact met u opnemen naar aanleiding van uw feedback?{' '}
-          <Optional>(niet verplicht)</Optional>
-        </StyledLabel>
-
-        <CheckboxWrapper inline htmlFor="allows-contact">
-          <Checkbox
-            data-testid="ktoAllowsContact"
-            id="allows-contact"
-            aria-describedby="subtitle-allows-contact"
-            name="allows-contact"
-            onChange={onChangeAllowsContact}
-          />
-          Ja
-        </CheckboxWrapper>
-      </GridArea>
+      {satisfactionIndication === 'nee' && (
+        <GridArea>
+          <Heading forwardedAs="h2">Contact</Heading>
+          <Paragraph id="subtitle-allows-contact">
```

Comment:
@kiruys I chose Heading h2 here and default Paragraph. In the design of invision it seems smaller but I chose default paragraph like in the design system.